source/jwtd/jwt_botan.d

```diff
@@ -33,6 +32,16 @@ version (UseBotan) {
 			sign = signer.signMessage(cast(const(ubyte)*)msg.ptr, msg.length, *rng)[].dup;
 		}
 
+		void sign_es(string emsaName) {
+			import botan.filters.data_src;
+			import botan.pubkey.algo.ecdsa;
+
+			Unique!AutoSeededRNG rng = new AutoSeededRNG;
+			auto privKey = loadKey(cast(DataSource)DataSourceMemory(key), *rng);
+			auto signer = PKSigner(privKey, emsaName, DER_SEQUENCE);
```

Comment:
DER_SEQUENCE is used to make the same signature format with OpenSSL (http://botan.randombit.net/manual/pubkey.html#_CPPv29PK_Signer)